.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  publish-npm:
+    name: Publish to npm
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    if: github.ref == 'refs/heads/master'
```

Reply:
@thomcsmits By adding this, publishing a new version will happen only when a tagged commit is pushed to `master`. This seems to be what we need?

Reply:
Yes, great!